include/morphio/enums.h

```diff
@@ -62,8 +62,8 @@ enum SectionType {
     SECTION_GLIA_ENDFOOT = 3,
 
     // All section types equal or above this number are custom types according
-    // to neuromorpho.org standard
-    SECTION_CUSTOM_START = 5,
+    // to neuromorpho.org standard (http://neuromorpho.org/StdSwc1.21.jsp)
```

Comment:
maybe there is something I did not get right. On the StdSwc1 they say: 
```
Codes 5-10 signify custom tags, typically defined in the header of the swc file when used.
In general, soma (tag 1) must start the cell and can only connect to itself. 
```
So the original  enums was ok to me no ?

Reply:
Yes, the problem is that MorphIO raises an error in case of custom tag. Should MorphIO raise if custom tags are allowed tags in morphology?

Reply:
Ah ok. Hum... From what I read the correct behavior should be : 
If the custom *is not* inside the header then we raise. Otherwise we just propagate the type. 
But the problem is : in section, the type is defined using SectionType which is an enum value to simplify the queries and the bindings. But here, we are dealing with values that should be evaluated at run time and this does not leave well with the enums.

So I don t know what to do ... Maybe having enum values :
```
SECTION_CUSTOM_5 = 5,
SECTION_CUSTOM_6 = 6,
...
SECTION_CUSTOM_10 = 10,
```
could make sense ?

Reply:
I don't think we should parse the header, I haven't seen that used, and until someone asks for it, let's skip it.

I think @tomdele suggestion of having the `SECTION_CUSTOM_5 = 5,`... is the best we can do.

Reply:
well, the description is vague (as always) : "Codes 5-10 signify custom tags, **typically defined** in the header of the swc file when used." (c). This **typically defined** confuses me currently. Ans what if it is not typically defined? :D

Reply:
**typically defined (c)** == do whatever you want...

Reply:
Also can you please give an example of .swc with header where custom type is defined. I've never seen how a proper header in .swc looks like.

Reply:
I think we should just create the custom tags : 
```
SECTION_CUSTOM_5 = 5,
SECTION_CUSTOM_6 = 6,
...
SECTION_CUSTOM_10 = 10,
```
without parsing the header and set the undefined to 11.

Reply:
ok, thank you so much for explanation!

Reply:
```
SECTION_OUT_OF_RANGE_START = 11
```
and raise if : 
```
type> SECTION_OUT_OF_RANGE_START 
```

Reply:
Should I still add the below in case we have `SECTION_OUT_OF_RANGE_START` ?
```
    SECTION_CUSTOM_5 = 5,
    SECTION_CUSTOM_6 = 6,
    SECTION_CUSTOM_7 = 7,
    SECTION_CUSTOM_8 = 8,
    SECTION_CUSTOM_9 = 9,
    SECTION_CUSTOM_10 = 10,
```

Reply:
yes I think so. If the user want to access the custom sections, he/she will still be able to do it using the enum (which is the default way of doing it).

Reply:
thank you